frontend/src/components/Notification.jsx

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from "react";
+import Avatar from "./Avatar.jsx";
+
+//images
+import avatar_image from "../images/placeholder-images/avatar.png";
+
+class Notification extends Component{
+
+    render(){
+        return(
+            <div className="notification">
```

Comment:
className="review-notification" is more acurate

Reply:
*The text on the right is not aligned properly. (vertically)*
By this you mean button on right ?

Reply:
I'm talking about added this article and avatar.